From 3df7ded403ac21e9309c9580c87dcc4f2fcea621 Mon Sep 17 00:00:00 2001
From: grotaclas <grotaclas@gmail.com>
Date: Sun, 04 May 2025 12:55:39 -0000
Subject: [PATCH] avoid permaclaims on sea tiles from GER and JAP missions

The missions jap_conquer_south_china (Conquer South China) and
JHC_GER_scramble_for_africa (Scramble for Africa) use
every_trade_node_member_province to award permanent claims
without checking if the member province is a sea tile

no bugreport, but a reddit post: https://www.reddit.com/r/eu4/comments/1jo4w60/i_think_the_german_mission_tree_gave_me_claims_on
---
 missions/DOM_Japanese_Missions.txt | 1 +
 missions/WOC_German_Missions.txt   | 3 +++
 2 files changed, 4 insertions(+)

diff --git a/missions/DOM_Japanese_Missions.txt b/missions/DOM_Japanese_Missions.txt
index 0be6da0f..6b007662 100755
--- a/missions/DOM_Japanese_Missions.txt
+++ b/missions/DOM_Japanese_Missions.txt
@@ -287,6 +287,7 @@ DOM_japanse_missions_1 = {
 							NOT = { is_core = ROOT } 
 							NOT = { province_id = 1926 }
 							NOT = { province_id = 1361 }
+							is_sea = no
 						}
 						add_permanent_claim = ROOT 
 					}
diff --git a/missions/WOC_German_Missions.txt b/missions/WOC_German_Missions.txt
index d2c9d54f..c781b60f 100755
--- a/missions/WOC_German_Missions.txt
+++ b/missions/WOC_German_Missions.txt
@@ -536,6 +536,7 @@ JHC_GER_2 = {
 						limit = {
 							NOT = { is_core = ROOT }
 							NOT = { is_permanent_claim = ROOT }
+							is_sea = no
 						}
 						add_permanent_claim = ROOT
 					}
@@ -545,6 +546,7 @@ JHC_GER_2 = {
 						limit = {
 							NOT = { is_core = ROOT }
 							NOT = { is_permanent_claim = ROOT }
+							is_sea = no
 						}
 						add_permanent_claim = ROOT
 					}
@@ -554,6 +556,7 @@ JHC_GER_2 = {
 						limit = {
 							NOT = { is_core = ROOT }
 							NOT = { is_permanent_claim = ROOT }
+							is_sea = no
 						}
 						add_permanent_claim = ROOT
 					}
-- 
2.45.2

